proxy/src/main/java/com/velocitypowered/proxy/connection/player/resourcepack/ModernResourcePackHandler.java

```diff
@@ -95,6 +97,20 @@ public void queueResourcePack(final @NotNull ResourcePackInfo info) {
     }
   }
 
+  @Override
+  public void queueResourcePack(@NotNull ResourcePackRequest request) {
+    if (request.packs().size() > 1) {
+      player.getConnection().write(new BundleDelimiterPacket());
```

Comment:
Have a single `INSTANCE` field in `BundleDelimiterPacket` instead of creating it each time?